src/copy-paste.js

```diff
@@ -0,0 +1,347 @@
+        }
+        var data = copy(),
+            mugs = vellum.getCurrentlySelectedMug(true);
+        mugs = _.filter(mugs, function (mug) { return mug.options.isCopyable; });
+        if (mugs && mugs.length) {
+            vellum.data.core.form.removeMugsFromForm(mugs);
+        }
+        return data;
+    }
+
+    function copy() {
+        if (!_.isUndefined(window.analytics)) {
+            window.analytics.track("Copy questions in form builder");
+        }
+        var mugs = vellum.getCurrentlySelectedMug(true),
+            seen = {};
+        if (!mugs || !mugs.length) { return ""; }
+
+        function serialize(mug) {
+            if (seen.hasOwnProperty(mug.ufid)) {
+                return;
+            }
+            seen[mug.ufid] = true;
+            if (!mug.options.isCopyable) {
+                return;
+            }
+            var row = mug.serialize(),
+                children = form.getChildren(mug);
+            _.each(row, function (value, key) {
+                if (!headings.hasOwnProperty(key)) {
+                    header.push(key);
+                    headings[key] = true;
+                }
+            });
+            if (children.length) {
+                row = [row].concat(_.map(children, serialize));
+            }
+            return row;
+        }
+
+        var headings = {id: true, type: true},
+            header = [],
+            form = mugs[0].form,
+            rows = _.filter(_.flatten(_.map(mugs, serialize)), _.identity);
+
+        if (rows.length === 0) {
+            return "";
+        }
+        header = ["id", "type"].concat(_.sortBy(header, headerKey));
+        return tsv.tabDelimit([PREAMBLE, header].concat(_.map(rows, function (row) {
+            return _.map(header, function (key) {
+                var val = row[key];
+                return stringify(_.isUndefined(val) ? null : val);
+            });
+        })));
+    }
+
+    function paste(data) {
+        if (!_.isUndefined(window.analytics)) {
+            window.analytics.track("Paste questions in form builder");
+        }
+        var next = tsv.makeRowParser(data);
+        if (!_.isEqual(next().slice(0, 2), PREAMBLE)) {
+            return ["Unsupported paste format"];
+        }
+        var types = vellum.data.core.mugTypes.allTypes,
+            form = vellum.data.core.form,
+            mug = vellum.getCurrentlySelectedMug(),
+            header = next(),
+            row = next(),
+            errors = new mugs.MugMessages(),
+            node = {id: null, mug: mug, parent: null},
+            into = {into: 1, last: 1},
+            later = [],
+            values, pos, parent;
+        errors.add = function (message) {
+            errors.update(null, {
+                key: message,
+                level: mugs.ERROR,
+                message: message
+            });
+        };
+        vellum.beforeBulkInsert(form);
+        for (; row; row = next()) {
```

Comment:
It could, but that produces lint error (which breaks the build): src/copy-paste.js: line 269, col 28, Expected a conditional expression and instead saw an assignment.